slice/internal/webhooks/jobset_webhook.go

```diff
@@ -55,22 +61,55 @@ func (r *JobSetWebhook) Default(ctx context.Context, obj runtime.Object) error {
 	log := ctrl.LoggerFrom(ctx).WithName("jobset-accelerator-gke-webhook")
 	log.V(5).Info("Applying defaults")
 
-	// handle webhooks for Kueue related JobSets
-	cq, ok := jobSet.Labels[kueueconstants.QueueLabel]
-	if !ok || cq == "" {
+	if jobSet.Labels[kueueconstants.QueueLabel] == "" {
 		return nil
 	}
 
 	for i, rj := range jobSet.Spec.ReplicatedJobs {
+		tpuTopology := rj.Template.Spec.Template.Annotations[TPUTopologyAnnotation]
```

Comment:
Pull out the inside of this for-loop to a function? something like `annotateReplicatedJobWithTopology`